.github/workflows/test.yml

```diff
@@ -57,19 +57,26 @@ jobs:
           echo "SDKROOT=$(xcrun --sdk macosx --show-sdk-path)" >> "${GITHUB_ENV}"
 
       # Caching
+      # Install gnu-tar because BSD tar is buggy
+      # https://github.com/actions/cache/issues/403
+      - name: Install GNU tar (Macos)
+        if: matrix.build == 'macos'
```

Comment:
Oops, I think this should be `if: matrix.os == 'macos-latest'`

Reply:
Ah thanks